pipelines/deployer/deployer.yaml

```
@@ -337,6 +338,44 @@ check_cloudwatch: &check_cloudwatch
       echo "FAIL: No logs have been detected reaching cloudwatch since $LOGS_SINCE"
       exit 1
 
+check-health-monitoring: &check-health-monitoring
+  platform: linux
+  image_resource: *task_image_resource
+  params:
+    ACCOUNT_ROLE_ARN: ((account-role-arn))
+    AWS_REGION: eu-west-2
+    AWS_DEFAULT_REGION: eu-west-2
+    CLUSTER_NAME: ((cluster-name))
+  run:
+    path: /bin/bash
+    args:
+      - -eu
+      - -opipefail
+      - -c
+      - |
+        echo "Assuming AWS deployer role..."
+        AWS_CREDS="$(aws-assume-role $ACCOUNT_ROLE_ARN)"
+        eval "${AWS_CREDS}"
+        echo "Fetching kubeconfig from aws..."
+        aws eks update-kubeconfig --name "${CLUSTER_NAME}" --kubeconfig ./kubeconfig
+        export KUBECONFIG=$(pwd)/kubeconfig
+        # Check Prometheus
+        echo "Port forwarding Prometheus to localhost:9090..."
+        kubectl -n gsp-system port-forward service/gsp-prometheus-operator-prometheus 9090:9090 &
+        sleep 5
+        echo "Curling forwarded port..."
+        curl --silent --show-error --max-time 5 --fail "http://127.0.0.1:9090" | grep "<title>Prometheus"
```

```suggestion
        curl --silent --show-error --max-time 5 --fail --location "http://127.0.0.1:9090" | grep "<title>Prometheus"
```

Needs the `-L`/`--location` option to follow the redirects.